instances/disease/malignantNeoplasm.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/disease/malignantNeoplasm",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Disease",
+  "definition": " A 'Malignant neoplasm' is composed of atypical, often pleomorphic cells that invade other tissues. Malignant neoplasms often metastasize to distant anatomic sites and may recur after excision [(adapted from [OLS ontology search, malignant neoplasm](https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C9305)].",
```

Comment:
+ "A 'malignant neoplasm' ..."
+ The reference is not really correct since the ontology definition is referencing already NCI
+ the definition needs to be optimized a bit...

Suggestion:
"A 'malignant neoplasm' is composed of atypical, often pleomorphic cells that uncontrollably grow and multiply, spreading into surrounding tissue and even invading distant anatomic sites (metastasis). Many malignant neoplasm form solid tumors, but cancers of the blood generally do not. [adapted from [NCI](https://www.cancer.gov/about-cancer/understanding/what-is-cancer)]"